src/odemis/dataio/tiff.py

```diff
@@ -1795,13 +1795,22 @@ def _saveAsMultiTiffLT(filename, ldata, thumbnail, compressed=True, multiple_fil
         # Only get the corresponding data for this file
         ldata = sorted_x[file_index][1]
 
+    # Extract ImageJ compatible Image description
+    num_channels = len(ldata)
+    try:
+        _, _, num_slices, height, width = ldata[0].shape
+    except:
+        num_slices = 1
+    imagej_description = (f"ImageJ=1.11a\nimages={num_slices * num_channels}\nchannels={num_channels}\nslices="
```

Comment:
I think it'd be nicer to read (and easier to extend) if you first make it a dict, and then convert it to a string. Something like that:
```
md = {
    "ImageJ": "1.11a",
    "images": num_slices * num_channels,
:
}
imagej_description = "\n".join(f"{k}={v}" for k, v in md) + "\n"
```